### PR TITLE
NetEase.InfiniteLagrange version 20221215

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92199)

### CODE REVIEW

**Comment by Trenly** [comment]
@msftbot zip

**Review by ItzLevvie — DISMISSED** [review]

**Comment by solomoncyj** [comment]
.

**Comment by Trenly** [comment]
> 2023-02-09 05:06:41.043 [REPO] Examining ARP entries for User | X86
2023-02-09 05:06:49.435 [CLI ] Caught std::exception: bad allocation

File: manifests/n/NetEase/InfiniteLagrange/20221215/NetEase.InfiniteLagrange.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NetEase.InfiniteLagrange
+PackageVersion: 20221215
+PackageLocale: en-US
+Publisher: NetEase
+PackageName: Infinite Lagrange
```
Comment (stephengillie) [inline manifests/n/NetEase/InfiniteLagrange/20221215/NetEase.InfiniteLagrange.locale.en-US.yaml:8]:
```suggestion
PackageName: Infinite Lagrange
PackageUrl: https://www.infinitelagrange.com/
```

**Review by Trenly — DISMISSED** [review]

**Comment by stephengillie** [comment]
The package installed normally, but gave an error when launched from the Start menu in a VM:
image

An internet search suggests the error might indicate a missing DLL. Does the package require a dependency?

**Comment by solomoncyj** [comment]
must be redist

**Review by stephengillie — APPROVED** [review]

**Pull request merged**